docker-compose-prod.yml

```diff
@@ -18,13 +18,19 @@ services:
       - database
 
   database:
-    image: postgres:13
+    image: mysql:5.7
+    restart: on-failure
+    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
     environment:
-      - POSTGRES_DB=ccm
-      - POSTGRES_USER=admin
-      - POSTGRES_PASSWORD=admin
+      - MYSQL_ROOT_PASSWORD=ccm_root_pw
+      - MYSQL_HOST=ccm_db_prod
+      - MYSQL_PORT=3306
+      - MYSQL_DATABASE=ccm
+      - MYSQL_USER=admin
+      - MYSQL_PASSWORD=admin
```

Comment:
Can we have same mysql property names as in `.env` file?

Reply:
I'm not sure what you're asking for here. These need to be set up this way in the MySQL container. I could change the database parameters in the application container so they use these names. Is that what you're asking for? I guess I slightly prefer not including the database flavor name in the variables, but could be convinced otherwise. I also think `db.name` is clearer in our application code than `db.database`.

https://github.com/tl-its-umich-edu/canvas-course-manager-next/blob/3f7536de69f87cbe80902473209f874aefb1e270/ccm_web/server/config.ts#L62-L67

Reply:
Yes! OK if you leave it off. I don't have a convincing reason but I looks like 2 different DB configuration options from face of it one for local and other for non-prod

Reply:
Hm, I'll follow up with you directly, I would like to understand your thinking here.

Reply:
Going to keep it as is for now.